api/tests/src/api/users/test_user_saved_opportunities_get.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from src.auth.api_jwt_auth import create_jwt_for_user
+from src.db.models.user_models import UserSavedOpportunity
+from tests.src.db.models.factories import OpportunityFactory, UserFactory
+
+
+@pytest.fixture
+def user(enable_factory_create, db_session):
+    user = UserFactory.create()
+    db_session.commit()
+    return user
```

Comment:
Factories commit when you call create, we can remove the commit and just do 
```suggestion
    return UserFactory.create()
```